### ISSUE TITLE
Add an FAQ page

### ISSUE THREAD

**Issue by iangilman** [openseadragon/openseadragon#2368]
@pearcetm mentioned it, and yes, seems like a good idea! 

For starters, let's collect some FAQs here... What do you think should go on it?

**Comment by iangilman** [comment]
I'm going to close https://github.com/openseadragon/site-build/issues/250 in favor of this issue, so I'm bringing over the comment from there: 

That might be the right place for the tile seam fix (see discussion at https://github.com/openseadragon/openseadragon/discussions/2085#discussioncomment-1957842), and I'm sure there are others.

**Comment by iangilman** [comment]
As mentioned in https://github.com/openseadragon/openseadragon/issues/2360, it might be nice to mention something about how you have to give the viewer some size or you won't see anything.

**Comment by pearcetm** [comment]
Issues with mobile portrait mode that are fixed by `<meta name="viewport" content="width=device-width, initial-scale=1" />` would be good to include (https://github.com/openseadragon/openseadragon/issues/2342#issuecomment-1524071957).

**Comment by iangilman** [comment]
Yes! Good one :)

**Comment by iangilman** [comment]
How about we get the FAQ started on the wiki here?

https://github.com/openseadragon/openseadragon/wiki/FAQ

Maybe we move it to the main site eventually, or maybe the wiki is a good place for it so it can be a bit more dynamic. Either way, we want to link to it from various places on the main site.

**Comment by pearcetm** [comment]
@iangilman I love the idea of using the wiki for this. I added the issues we've discussed so far. To make it most useful, it'll be important to link to it from somewhere (or multiple places) on the openseadragon.github.io page, as well as in the issues forum here.

**Comment by iangilman** [comment]
@pearcetm Fantastic! Beautifully written up! 

I'll go about adding it to the main site and otherwise announcing it. I suppose we could do that thing where some info shows up whenever you try to create a new issue and link from there. Do you offhand know how one does that? If not, I'll hunt around...

**Comment by iangilman** [comment]
Okay, I've updated the main site: 

https://openseadragon.github.io/#support

... As well as the docs page (and added a mention to the viewer needing width and height):

https://openseadragon.github.io/docs/

Here's the PR for that:

https://github.com/openseadragon/site-build/pull/281

I've also tweeted:

https://twitter.com/openseadragon/status/1672009353526923265

... And shared on Discord. Hopefully people know about the FAQ now! 

I'm closing this issue now. Thank you for making it happen @pearcetm!
